jomar-restrepo/tarea2/styles.css

```
@@ -0,0 +1,101 @@
+.fondo {
+    background-image: url("/jomar-restrepo/tarea2/img/fondop1.jpg");
```

@Leviathan9020 , you should use relative paths

```suggestion
    background-image: url("../img/fondop1.jpg");
```